locale/en/locale.cfg

```diff
@@ -32,12 +32,11 @@ no-energy=Not enough energy to teleport.
 teleported=Teleported: __1__ to __2__ __3__
 teleport-failed=Unable to teleport: __1__ to __2__. [color=#ff0000]Suitable position not found[/color] __3__.
 
-[coe-launch-gui]
-button_show_statistics=Show Cities of Earth Statistics
-title_coe_ui=Cities of Earth Statistics
-button_close=Close
+[coe-stats-gui]
+button_statistics=Show Cities of Earth Statistics
```

Comment:
Some word I can't think of that means this doesn't offer enough context.
`button_show_statistics_tooltip`

Reply:
I do not understand.

Reply:
Ahh the key isn't verbose enough (thats the word I was looking for)

`button_statistics_tooltip=Show Cities of Earth Statistics`